audb/core/load.py

```diff
@@ -1015,3 +1057,96 @@ def load_media(
         media = [audeer.replace_file_extension(m, format) for m in media]
 
     return [os.path.join(db_root, m) for m in media]
+
+
+def load_table(
+        name: str,
+        table: str,
+        *,
+        version: str = None,
+        cache_root: str = None,
+        num_workers: typing.Optional[int] = 1,
+        verbose: bool = True,
+) -> pd.DataFrame:
+    r"""Load a database table.
+
+    If you are only interested in an original table
```

Comment:
I'm not so happy with "original table" here. If you use `audb.load()` with `full_path=True` and do not specify a specific `format` or filter for `media`, you would get the same table. I'd rather highlight that you can directly get the content of a table with this function. We should probably mention that media files are not downloaded, so that all or some of the media may not exist on disk.

Reply:
I updated it to

![image](https://user-images.githubusercontent.com/173624/117246590-3166f080-ae3d-11eb-9df6-1c859859425a.png)

I also adjusted the docstring of `load_media()` accordingly

![image](https://user-images.githubusercontent.com/173624/117246635-45aaed80-ae3d-11eb-9c05-26e0ffa3f54c.png)